src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionParsing.scala

```diff
@@ -113,34 +116,43 @@ trait SlickExpressionParser extends JavaTokenParsers {
   private def entityRootDot: Parser[RootFunc] =
     "this." ^^ { _ => entityRootFunc}
 
+  private val namespacedIdent: Parser[String] = rep1sep(ident, ":") ^^ {
```

Comment:
Update: works now with `Parser[AttributeName]` and throws a `RawlsException` for `one:two:three`